app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  
+  def new
+    redirect_to home_path if logged_in?
```

Comment:
You might want to do `redirect_to home_path and return if logged_in?`. This will make it so `@user = User.new` is not hit every time an authenticated user hits this route. Not a big deal now though.